rotary/rules/rotary.smk

```diff
@@ -20,20 +21,15 @@ DB_DIR_PATH = config.get('db_dir')
 sample_tsv_path = 'samples.tsv'
 SAMPLES = parse_sample_tsv(sample_tsv_path)
 
-first_sample = next(iter(SAMPLES.values()))
-
-# sample and reads
-SAMPLE_ID = first_sample.identifier
-LONG = first_sample.long_read_path
-SHORT_R1 = first_sample.short_read_left_path
-SHORT_R2 = first_sample.short_read_right_path
+SAMPLE_NAMES = list(SAMPLES.keys())
 
 # Specify the minimum snakemake version allowable
 min_version("7.0")
 # Specify shell parameters
 shell.executable("/bin/bash")
 shell.prefix("set -o pipefail; ")
 
+POLISH_WITH_SHORT_READS = True
```

Comment:
Thanks for adding this temporary workaround. I agree, we'll need to rework the logic for short read polishing to make it work with the multi-sample workflow

Reply:
@LeeBergstrand For now, I think the simplest solution would be to lock a single rotary run to one run mode (with short read polishing or without). The run mode can be set based on if R1 and R2 files are in the samples.tsv file or not. Do you think this might be a reasonable target for the MVP version of rotary? (It would only take minor modifications of the current code.)

Longer-term, it would be great if we could make rotary auto-determine whether or not to use short reads on a per-sample basis. Basically, the DAG will need to go in different directions depending on the sample, and this should ideally be defined at the start of execution of the pipeline. I am not sure how to do this. The closest thing I know how to do is to set up snakemake checkpoints that dynamically adjust the DAG partway through the pipeline, based on outputs of some commands. I think these snakemake checkpoints should generally be avoided when possible, because (I think) they cause the parts of the DAG downstream from the checkpoint to be hidden (e.g., when `--dry-run` is used). It might take some time to figure out an ideal solution...

Reply:
@jmtsuji I'll have to check if the sample object still enforces having both short and long reads during its instantiation.

Reply:
@jmtsuji My main concern is that some samples have short reads and others don't. You would have to solve that with by-sample switching logic, which is a rewrite. Is that situation common? I am leaning towards putting a flag in the config for now as the MVP and getting things merged into development so you can start finalizing the Illumina QC.

Reply:
@LeeBergstrand I share your concern -- I imagine it will be fairly common that some genomes will have short reads and some won't. For now, I think the easiest MVP solution is to switch on a per-run basis between short read vs. no short read mode, e.g., via a CLI flag, like you mentioned. But longer-term, I definitely think we should try to code some by-sample switching logic.

Reply:
Update: I saw that you made the multi pathing branch -- thanks!!

Reply:
@jmtsuji I will leave it as a config variable for now for the near-term MVP. We can add it as a command-line flag or other methods once we have everything merged into develop.